templates/group.html

```diff
@@ -1 +1 @@
-<a ng-href="#/{{ctrl.group}}">{{ctrl.group}} ({{ctrl.groupCount}})</a>
+<a ng-href="#/{{ctrl.group}}">{{ ctrl.group }} ({{ ctrl.groupCount | counterFormatter }})</a>
```

Comment:
Better, but you need to use the li element
`<li class="app-navigation-entry-utils-counter">{{ count | counterFormatter }}</li>`

![capture d ecran_2017-01-08_17-43-40](https://cloud.githubusercontent.com/assets/14975046/21751599/049c8916-d5ca-11e6-8645-ffa8cc7e18f7.png)